ansible/server-state-playbooks/nightshade-web/playbook.yml

```diff
@@ -42,18 +23,35 @@
       lvm_lvfilesystem: "{{ provision_root_filesystem }}"
 
     # OMERO.web configuration in host_vars in different repository
```

Comment:
So `omero_web_config_set` is in hostvars, but `omero_server_config_set` is in `ome-dundeeomero/playbook.yml`?